packages/p2p/src/components/modal-manager/modal-manager-context-provider.jsx

```diff
@@ -57,6 +57,9 @@ const ModalManagerContextProvider = props => {
     const isCurrentModal = (...keys) => keys.includes(active_modal.key);
 
     const showModal = modal => {
+        // // if there are multiple of the same modals being called at the same time
+        if (Object.keys(active_modal).length > 0 && active_modal.key === modal.key) return;
+
```

Comment:
This checks if the same modal is being called multiple times, thus do not show or stack the same modal, otherwise the following may occur:
<img width="445" alt="Screenshot 2023-02-15 at 11 42 16 AM" src="https://user-images.githubusercontent.com/103016120/218925434-200a8c7d-9d8f-4e7b-aa48-d892e5c0f417.png">